frontend/src/lib/v2/WorkflowUtils.ts

```diff
@@ -48,7 +49,9 @@ export function isTemplateV2(templateString: string): boolean {
 
 // Assuming template is the JSON format of PipelineSpec in api/v2alpha1/pipeline_spec.proto
 export function convertYamlToV2PipelineSpec(template: string): PipelineSpec {
-  const pipelineSpecYAML = jsyaml.safeLoad(template);
+  // If pipeline_spec exists in the return value of safeload, which means the original yaml contains
+  // platform_spec, then the pipeline spec is stored in pipeline_spec field.
```

Comment:
This is a temporary solution which omit platform-specific proto. However, KFP frontend should support kubernetes platform-specific proto because it contains information that defines this pipeline. Recommend creating a bug to follow up on this item.

Reply:
/cc @chensun

Reply:
James was correct that we need to read in kubernetes specific config from the `platform_spec` section. This would be needed to support volume for example:
https://github.com/kubeflow/pipelines/blob/6a55449ffb5b3d935b32033dc9a13b1852c059a4/kubernetes_platform/python/test/snapshot/data/create_mount_delete_dynamic_pvc.yaml#L180-L181